tests/lib/snaps/test-snapd-policy-app-consumer/meta/snap.yaml

```diff
@@ -353,6 +353,9 @@ apps:
   polkit:
     command: bin/run
     plugs: [ polkit ]
+  posix-mq:
+    command: bin/run
+    plugs: [ posix-mq ]
```

Comment:
is this enough? also won't this give us trouble as the feature is parser dependent?

Reply:
@mvo5 added this - I'm not sure

Reply:
It was fine in my local run but maybe I did something else wrong. In any case it unblocks spread enough that we see some more failures from the static tests, notably https://github.com/snapcore/snapd/runs/5885786791?check_suite_focus=true#step:19:255 which looks like some mocking is missing and a bit below https://github.com/snapcore/snapd/runs/5885786791?check_suite_focus=true#step:19:283

Reply:
I added a call to `apparmor_sandbox.MockFeatures()` so it no longer fails the tests due to the missing feature in `apparmor_parser`